self-api/src/main/java/com/selfxdsd/api/Language.java

```diff
@@ -104,7 +128,95 @@ public final String categorize(final String command) {
      * @return String reply or null if nothing is found.
      */
     public final String reply(final String key) {
-        return this.responses.getProperty(key);
+        final String reply = this.responses.getProperty(key);
+        return tryReplyFromLink(reply);
```

Comment:
@criske just ``return followPossibleLink(this.responses.getProperty(key));`` should be enough.